pybikes/gbfs.py

```diff
@@ -0,0 +1,71 @@
+# -*- coding: utf-8 -*-
+# Copyright (C) 2010-2012, eskerda <eskerda@gmail.com>
+# Distributed under the AGPL license, see LICENSE.txt
+
+import json
+
+from .base import BikeShareSystem, BikeShareStation
+from . import utils
+
+__all__=['Gbfs', 'GbfsStation']
+
+class Gbfs(BikeShareSystem):
+
+    def __init__(self, tag, meta, feed_url):
+        super(Gbfs, self).__init__(tag, meta)
+        self.feed_url=feed_url
```

Comment:
Can we make feed_url available in meta here so we can expose it later on the API?

Something like

```
meta['gbfs_href'] = feed_url
```